trending_deploy/deploy.py

```diff
@@ -1,32 +1,95 @@
-
-from typing import List, Literal
-
-from trending_deploy.constants import Model
+from typing import List, Literal, Dict, Any
+import os
+from huggingface_hub import (
+    InferenceClient,
+    create_inference_endpoint, 
+    get_inference_endpoint,
+    list_inference_endpoints,
+    model_info as get_model_info
+)
+
+from trending_deploy.constants import Model, MEMORY_USAGE_TO_INSTANCE, Instance, INSTANCES
+from trending_deploy.models import get_num_parameters_from_model, get_viable_instance_from_num_parameters
+
+# Configuration
+HF_TOKEN = os.environ.get("HF_TOKEN")
+VENDOR = "aws"
+REGION = "us-east-1"
+TYPE = "open"
```

Comment:
I think this needs to be "public":
>  type ([`InferenceEndpointType], *optional*) -- The type of the Inference Endpoint, which can be “protected”(default),“public”or“private”`. 

https://huggingface.co/docs/huggingface_hub/v0.30.2/en/package_reference/hf_api#huggingface_hub.HfApi.create_inference_endpoint

Reply:
fixed